client/interactions.lua

```diff
@@ -105,6 +71,7 @@ local function handcuffedEscorted()
     end
     lib.requestAnimDict('mp_arresting')
     TaskPlayAnim(cache.ped, 'mp_arresting', 'idle', 8.0, -8, -1, cuffType, 0, false, false, false)
+    RemoveAnimDict('mp_arresting')
```

Comment:
lib.playAnim has the same parameters as TaskPlayAnim, but does the request and remove.
```suggestion
    lib.playAnim(cache.ped, 'mp_arresting', 'idle', 8.0, -8, -1, cuffType, 0, false, false, false)
```

Reply:
I'd prefer to put it in the next PR so as not to confuse more